emails/views.py

```diff
@@ -336,11 +341,41 @@ def _sns_message(message_json):
     return ses_relay_email(from_address, relay_address, subject, message_body)
 
 
+def _get_attachment_metrics(part):
+    incr_if_enabled('email_with_attachment', 1)
+    fn = part.get_filename()
+    ct = part.get_content_type()
+    payload = part.get_payload(decode=True)
+    payload_size = len(payload)
+    if fn:
+        extension = os.path.splitext(fn)[1]
+    else:
+        extension = mimetypes.guess_extension(ct)
+    logger.error(
+        'Attachment found in email',
+        extra={
+            'content-type': ct,
+            'extension': extension,
+            'payload-size': payload_size
+        }
+    )
+    tag_type = 'attachment'
+    attachment_extension_tag = generate_tag(tag_type, ct)
+    attachment_content_type_tag = generate_tag(tag_type, extension)
```

Comment:
Oh wait, did this variable names get reversed somehow? `attachment_extension_tag` is based on `ct` while `attachment_content_type_tag` is based on `extension`? I may have messed that up in my suggestions. :( 

```suggestion
    attachment_extension_tag = generate_tag(tag_type, extension)
    attachment_content_type_tag = generate_tag(tag_type, ct)
```